tls: add ability to get cert/peer cert as X509Certificate object (#37070)
nodejs/node: doc/api/crypto.md

The issuer identification included in this certificate.

> ### `x509.issuerCert`

Reviewer: Given other API names such as `getPeerX509Certificate` and `getX509Certificate`, I have a slight preference not to abbreviate this name and to use `issuerCertificate` instead.

<!-- YAML
added: REPLACEME
-->

> * Type: {X509Certificate}
> 
> The issuer certificate (if known). Will be `undefined` if the issuer
> certificate is not available.

Reviewer 2: This suggests a difference between "not known" and "not available". I imagine they're the same thing, though?

```suggestion
* Type: {X509Certificate|undefined}

The issuer certificate. if the issuer certificate is not known, this property is
`undefined`.
```
